### PR TITLE
BZ1862098 - removing etcd DNS records from bare metal

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1862098

@jianlinliu, will you PTAL?

### CODE REVIEW

File: modules/installation-dns-user-infra.adoc [hunk 1]
```diff
@@ -49,27 +49,4 @@ machines that run the Ingress router pods, which are the worker nodes by
 default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 
-.2+|etcd
```
Comment (codyhoag) [inline modules/installation-dns-user-infra.adoc:52]: The removal looks good. The only thing I would suggest is possibly editing the column widths. Now with the etcd stuff removed, the `Record` and `Description` columns could be adjusted for readability. Maybe something like `[cols="1a,3a,5a",options="header"]`?

**Review by codyhoag — COMMENTED** [review]
One suggestion for column widths. All else looks good!

File: modules/installation-dns-user-infra.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/installation-dns-user-infra.adoc:52]: That does look good. Thanks!

**Comment by kalexand-rh** [comment]
This change was approved on the source bug. Merging.

**Pull request merged**